Fixes

TrySound opened the pull request.

TrySound committed:
fix #1

TrySound committed:
Added more info

TrySound committed:
fix #2

jakiestfu commented on bower.json:
```diff
@@ -1,6 +1,32 @@
+  "main": [
+    "src/ripple.js",
+    "src/ripple.css"
+  ],
```
Even though the source files happen to be a 1-1 to uncompressed dist files, I'd still prefer to reference the files in the dist folder instead. That being said, if we ever decide to use less as a src, this approach will not work. Thoughts?

Also, we'll need to update to 1.2.1 :)

TrySound replied:
So you may do non-minified and minified in dist folder. And both with your header.

TrySound replied:
I do not use grunt (gulp lover), so don't know how to do that.

TrySound committed:
No need sync. Bower version is current release tag.
And don't forget to publish package in bower.

TrySound committed:
prefix optimization
-ms- no need because animation was never prefixed in IE. In transforms, because prefix need in ie9, which doesn't support animations.
And nobody uses so old Mozilla

TrySound committed:
Version update and grunt fix

jakiestfu commented on Gruntfile.js:
```diff
@@ -60,17 +60,6 @@ module.exports = function(grunt) {
-    });
-
-    grunt.registerTask('default', ['jshint', 'uglify', 'cssmin', 'sync_versions']);
-    grunt.registerTask('develop', ['jshint', 'uglify', 'cssmin', 'sync_versions', 'watch']);
```
I can handle the additional grunt stuff, but why did you delete this?

TrySound replied:
Bower use auto version by github tags. So this is unnecessary. And bower must contain more info. And I add this info.

I understand grunt just know))
